core/src/main/java/io/lionweb/lioncore/java/language/LionCoreBuiltins.java

```diff
@@ -38,30 +47,61 @@ private LionCoreBuiltins() {
   }
 
   public static LionCoreBuiltins getInstance() {
```

Comment:
For me it's more intuitive to have several overloads of the same method next to each other.

Reply:
Right, I now grouped overloaded methods

Reply:
I have also done that in LionCore